even-more-fish-plugin/src/main/java/com/oheers/fish/config/messages/Message.java

```diff
@@ -186,17 +186,13 @@ public void broadcast(@NotNull final CommandSender sender, final boolean doColou
      *
      * @return The string from config that matches the value of id.
      */
-    public String getString(String normal, String... ids) {
-        for (String id : ids) {
-            if (!Messages.getInstance().getConfig().contains(id)) continue;
-            String returning = Messages.getInstance().getConfig().getString(id);
-            if (returning != null) return returning;
-            else {
-                EvenMoreFish.logger.log(Level.SEVERE, "No valid value in messages.yml for: " + id + " using default value instead.");
-                return normal;
-            }
+    public String getString(String normal, String id) {
+        String string = Messages.getInstance().getConfig().getString(id, null);
+        if (string == null) {
+            EvenMoreFish.logger.log(Level.SEVERE, "No valid value in messages.yml for: " + id + " using default value instead.");
```

Comment:
You can use a convinience function here. `EvenMoreFish.getInstance().getLogger().severe("No valid value in messages.yml for: " + id + " using default value instead.")`. Instead of specifying the level.